src/styles/blocks/header.scss

```diff
@@ -0,0 +1,83 @@
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  align-items: center;
+  outline: none;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  transition: all 300ms ease;
+  display: flex;
+  line-height: 60px;
+
+  &__item {
+    list-style-type: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    color: $nav-link;
+    font-size: $font-size;
+    height: 60px;
+    font-weight: 500;
+    text-transform: uppercase;
+  }
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline-start: 0;
+  margin: 0;
+  padding: 0;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should avoid using shorthand properties for margin and padding when only one value is being set. It's better to use `margin: 0;` and `padding: 0;` separately for clarity.